lib/ansible/module_utils/cloudfront.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2017 Willem van Ketwich
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Author:
+#   - Willem van Ketwich <willem@vanketwich.com.au>
+#
+# Common functionality to be used by the modules:
+#   - cloudfront_distribution
+#   - cloudfront_invalidation
+#   - cloudfront_origin_access_identity
+
+
+class CloudFrontHelpers:
```

Comment:
This really doesn't need to be an object - I'd prefer to have `from module_utils.cloudfront import helpers` to instantiating a class.